TODO: send upstream (in Gentoo, we don't rename it to cpp-httplib, so need to check for that as a library too)
--- a/cmake/FindCppHttplib.cmake
+++ b/cmake/FindCppHttplib.cmake
@@ -12,7 +12,7 @@ else()
     if(NOT "${CMAKE_MATCH_0}" STREQUAL "" AND "${_cpphttplib_version_string}" VERSION_GREATER_EQUAL "${CppHttplib_FIND_VERSION}")
       # Some dists like Fedora package cpp-httplib as a single header while some
       # dists like Debian package it as a traditional library.
-      find_library(CPPHTTPLIB_LIBRARY cpp-httplib)
+      find_library(CPPHTTPLIB_LIBRARY httplib)
       if(CPPHTTPLIB_LIBRARY)
         message(STATUS "Using system CppHttplib (${CPPHTTPLIB_LIBRARY})")
         add_library(dep_cpphttplib UNKNOWN IMPORTED)
